pkg/util/kubeadm/init.go

```diff
@@ -212,6 +213,16 @@ func NewInitCMD(out io.Writer, edgeConfig *cmd.EdgeadmConfig) *cobra.Command {
 			return err
 		}
 
+		// set edgeadm crisocket for runtime
+		switch edgeConfig.ContainerRuntime {
+		case constant.ContainerRuntimeDocker:
+			initOptions.externalInitCfg.NodeRegistration.CRISocket = constant.DefaultDockerCRISocket
+		case constant.ContainerRuntimeContainerd:
+			initOptions.externalInitCfg.NodeRegistration.CRISocket = constant.DefaultContainerdCRISocket
```

Comment:
done

Reply:
Can you also modify the document？ @malc0lm

Reply:
Document has been modified in README.md.